release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -87,6 +87,60 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-networking"]
 === Networking
 
+[id="ocp-4-13-networking-metrics"]
+==== Enhancements to networking metrics
+===== egress_ips_rebalance_total
+* Metric name: `ovnkube_master_egress_ips_rebalance_total`
+* *Help message:* `The total number of times assigned egress IP(s) needed to be moved to a different node.`
+
```

Comment:
I didn't notice this earlier. But, having _Metric name_ not bolded and _Help message_ bolded looks a bit odd to me.

Reply:
Hi @mburke5678 so I didn't know what to do here. For context, *Help message* is a UI element and the metric name isn't. In that case do you think it is ok to leave it bolded?

Reply:
@JoeAldinger Just leave it, I guess. Maybe I'm being too picky.